### ISSUE TITLE
Getting error when trying to install the Traefik using Helm

### ISSUE THREAD

**Issue by ahsanmirxa** [traefik/traefik-helm-chart#780]
### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik-helm-chart/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.traefik.io) and didn't find any.

### What version of the Traefik's Helm Chart are you using?

20.8.0

### What version of Traefik are you using?

2.9.6

### What did you do?

I tried uninstalling the traefik using Helm but resulted in error. 

### What did you see instead?

Error: failed to install CRD crds/kustomization.yaml: resource mapping not found for name: "" namespace: "" from "": no matches for kind "Kustomization" in version "[kustomize.config.k8s.io/v1beta1](http://kustomize.config.k8s.io/v1beta1)" 
ensure CRDs are installed first

### What is your environment & configuration?

AKS Cluster - v1.24.6
Helm - v3.9.3

Nothing special configuration to deploy

### Additional Information

```shell
None
```

**Comment by mloiseleur** [comment]
Thanks for your report @ahsanmirxa 

What is exactly your issue, because : 
1. title says _"error when trying to install"_
2. content says _"I tried uninstalling the traefik"_

Also, the `crds/kustomization.yaml` is not part of this Helm Chart, only a part of this repo : [.helmignore](https://github.com/traefik/traefik-helm-chart/blob/master/traefik/.helmignore)

Can you be more specific and provides more details with commands, results seen and results expected ?

**Comment by ahsanmirxa** [comment]
Yes, I was trying to install by cloning the repo locally, using Helm commands (helm upgrade -i traefik traefik/ (cloned chart folder), I figured out that kustomization.yaml is part of the repo and was perfectly working fine with it on AKS clusters v1.23.x,  but not on v1.24.x clusters.

Once I removed the kustomization.yaml from the crds, it happened on v1.24.x as well.

**Comment by mloiseleur** [comment]
I was able to reproduce your behavior _only_ by deleting the `.helmignore` file:
```bash
$ rm traefik/.helmignore
$ helm upgrade -i traefik ./traefik 
Release "traefik" does not exist. Installing it now.
Error: failed to install CRD crds/kustomization.yaml: resource mapping not found for name: "" namespace: "" from "": no matches for kind "Kustomization" in version "kustomize.config.k8s.io/v1beta1"
ensure CRDs are installed first
```

And I was able to fix it by restoring `.helmignore` file:
```bash
$ git checkout traefik/.helmignore 
Updated 1 path from the index
$ helm upgrade -i traefik ./traefik
Release "traefik" does not exist. Installing it now.
NAME: traefik
LAST DEPLOYED: Mon Jan 16 15:32:00 2023
NAMESPACE: default
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
Traefik Proxy v2.9.6 has been deployed successfully
on default namespace !
```

So in summary:
- `.helmignore` seems already present and configured as expected. 
- There is no (at least obvious) link between AKS and files ignored by Helm.

**Comment by ahsanmirxa** [comment]
I am not sure how did I missed the .helmignore when I cloned it from the repo.  I possibly missed/deleted it in moving the cloned folders between machines. Although I did manage it by deleting the kustomization.yaml itself. We are safe to close this. Thanks anyway for your time and effort!!
